libdd-profiling-ffi/src/exporter.rs

```diff
@@ -194,62 +180,6 @@ unsafe fn into_vec_files<'a>(slice: Slice<'a, File>) -> Vec<exporter::File<'a>>
         .collect()
 }
 
-/// If successful, builds a `ddog_prof_Exporter_Request` object based on the
-/// profile data supplied. If unsuccessful, it returns an error message.
-///
-/// For details on the `optional_internal_metadata_json`, please reference the Datadog-internal
-/// "RFC: Attaching internal metadata to pprof profiles".
-/// If you use this parameter, please update the RFC with your use-case, so we can keep track of how
-/// this is getting used.
-///
-/// For details on the `optional_info_json`, please reference the Datadog-internal
-/// "RFC: Pprof System Info Support".
-///
-/// # Safety
-/// The `exporter`, `optional_additional_stats`, and `optional_endpoint_stats` args should be
-/// valid objects created by this module.
-/// NULL is allowed for `optional_additional_tags`, `optional_endpoints_stats`,
-/// `optional_internal_metadata_json` and `optional_info_json`.
-/// Consumes the `SerializedProfile`
-#[no_mangle]
-#[must_use]
-#[named]
-pub unsafe extern "C" fn ddog_prof_Exporter_Request_build(
```

Comment:
I believe none of the dd-trace-* libraries are using `files_to_export_unmodified` field. So dropping this shouldn't be a problem, but still is a breaking change to note.